GameData/RealismOverhaul/RO_SuggestedMods/KerbalFoundries_2/KF-RO.cfg

```diff
@@ -0,0 +1,438 @@
+        {
+            key = 0.0  2.0 0.0 0.0
+            key = 7.8  1.5 0.0 0.0
+            key = 15.6 0.5 0.0 0.0
+            key = 23.5 0.0 0.0 0.0
+        }
+    }
+}
+
+@PART[KF-TrackLong]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    @mass = 0.95
+    @crashTolerance = 18
+
+    @MODULE[ModuleWheelBase] { @mass = 0.95 }
+    @MODULE[ModuleWheelMotor]
+    {
+        %wheelSpeedMax = 23.5
+        @idleDrain = 0
+
+        @RESOURCE[ElectricCharge] { @rate = 30 }
+
+        !torqueCurve,*{}
+        torqueCurve
+        {
+            key = 0.0  2.0 0.0 0.0
+            key = 7.8  1.5 0.0 0.0
+            key = 15.6 0.5 0.0 0.0
+            key = 23.5 0.0 0.0 0.0
+        }
+    }
+}
+
+@PART[KF-TrackMedium]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    @mass = 0.2
+    @crashTolerance = 18
+
+    @MODULE[ModuleWheelBase] { @mass = 0.2 }
+    @MODULE[ModuleWheelMotor]
+    {
+        %wheelSpeedMax = 23.5
+        @idleDrain = 0
+
+        @RESOURCE[ElectricCharge] { @rate = 15 }
+
+        !torqueCurve,*{}
+        torqueCurve
+        {
+            key = 0.0  2.0 0.0 0.0
+            key = 7.8  1.5 0.0 0.0
+            key = 15.6 0.5 0.0 0.0
+            key = 23.5 0.0 0.0 0.0
+        }
+    }
+}
+
+@PART[KF-TrackMole]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    @mass = 2
+    @crashTolerance = 18
+
+    @MODULE[ModuleWheelBase] { @mass = 2 }
+    @MODULE[ModuleWheelMotor]
+    {
+        %wheelSpeedMax = 23.5
+        @idleDrain = 0
+
+        @RESOURCE[ElectricCharge] { @rate = 100 }
+
+        !torqueCurve,*{}
+        torqueCurve
+        {
+            key = 0.0  2.0 0.0 0.0
+            key = 7.8  1.5 0.0 0.0
+            key = 15.6 0.5 0.0 0.0
+            key = 23.5 0.0 0.0 0.0
+        }
+    }
+}
+
+@PART[KF-TrackSimple]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    @mass = 0.065
+    @crashTolerance = 18
+
+    @MODULE[ModuleWheelBase] { @mass = 0.065 }
+    @MODULE[ModuleWheelMotor]
+    {
+        %wheelSpeedMax = 23.5
+        @idleDrain = 0
+        @RESOURCE[ElectricCharge] { @rate = 2 }
+
+        !torqueCurve,*{}
+        torqueCurve
+        {
+            key = 0.0  2.0 0.0 0.0
+            key = 7.8  1.5 0.0 0.0
+            key = 15.6 0.5 0.0 0.0
+            key = 23.5 0.0 0.0 0.0
+        }
+    }
+}
+
+@PART[KF-ScrewDrive]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    @mass = 1.0
+    @crashTolerance = 18
+
+    @MODULE[ModuleWheelBase] { @mass = 1.0 }
+    @MODULE[ModuleWheelMotor]
+    {
+        %wheelSpeedMax = 23.5
+        @idleDrain = 0
+
+        @RESOURCE[ElectricCharge] { @rate = 25 }
+
+        !torqueCurve,*{}
+        torqueCurve
+        {
+            key = 0.0  2.0 0.0 0.0
+            key = 7.8  1.5 0.0 0.0
+            key = 15.6 0.5 0.0 0.0
+            key = 23.5 0.0 0.0 0.0
+        }
+    }
+}
+
+@PART[KF-ALG-Small]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    @mass = 0.1
+    @maxTemp = 2000
+    %skinMaxTemp = 2000
```

Comment:
That's very high, and probably should just be subject to the normal RO limits.  Same for the rest of the parts here.

Reply:
Im pretty sure there was a reason i made them able to get hotter, but cant remember now, ill just default it